NetGrpcPrometheus/ClientInterceptor.cs

```diff
@@ -12,9 +12,8 @@ namespace NetGrpcPrometheus
     /// <summary>
     /// Interceptor for intercepting calls on client side
     /// </summary>
-    public class ClientInterceptor : Interceptor, IDisposable
+    public class ClientInterceptor : Interceptor
     {
-        private readonly MetricServer _metricServer;
```

Comment:
Hmm, I just noticed that the ctor taking an endpoint creates a MetricServer. It is, however, not assigned to this readonly field. So probably that is the mistake and it is no longer unused code?